bitex/api/REST/rest.py

```diff
@@ -81,10 +82,14 @@ def _query(self, method_verb, **request_kwargs):
         :param request_kwargs: kwargs for request.Request()
         :return: request.Response() object
         """
+        # self.proxies = {"http": "http://127.0.0.1:1087", "https": "http://127.0.0.1:1087"}
+        # print(method_verb,request_kwargs)
         if 'method' not in request_kwargs:
-            resp = requests.request(method_verb, **request_kwargs, timeout=self.timeout)
+            resp = requests.request(method_verb, **request_kwargs, proxies=self.proxies,
+                                    timeout=self.timeout)
         else:
-            resp = requests.request(**request_kwargs, timeout=self.timeout)
+            resp = requests.request(**request_kwargs, proxies=self.proxies, timeout=self.timeout)
+        # if not resp.ok: print('===RESP not OK',resp.ok, resp.reason, resp.status_code, resp.url)  # resp.content,resp.json())
```

Comment:
Please delete the debugging leftovers :)

Reply:
ok